section3/exercises/hashes.rb

```diff
@@ -1,59 +1,89 @@
 # In the below exercises, write code that achieves
 # the desired result. To check your work, run this
-# file by entering the following command in your terminal: 
+# file by entering the following command in your terminal:
 # `ruby section3/exercises/hashes.rb`
 
+
+puts '-' * 10 + "PART 1" + '-' * 10
+
 # Example: Write code that prints a hash holding grocery store inventory:
 foods = {apples: 23, grapes: 507, eggs: 48}
 p foods
 
 # Write code that prints a hash holding zoo animal inventory:
-zoo = #YOUR CODE HERE
+zoo = {bear: 2, meerkat: 5, eagle: 10}
 p zoo
 
-# Write code that prints all of the 'keys' of the zoo variable 
+# Write code that prints all of the 'keys' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+zoo.each do |zoo, animal|
+  puts "#{zoo}"
+end
 
-# Write code that prints all of the 'values' of the zoo variable 
+# Write code that prints all of the 'values' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+zoo.each do |zoo, animal|
+  puts "#{animal}"
+end
 
-# Write code that prints the value of the first animal of the zoo variable 
+# Write code that prints the value of the first animal of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo['bear']
```

Comment:
I think this would need to be `puts zoo[:bear]`. And to add another key-value pair, something like `zoo[:whale] = 1`.